otsdaq/NetworkUtilities/TCPServerBase.cc

```
@@ -217,7 +218,8 @@ void TCPServerBase::closeClientSocket(int socket)
 		{
 			try
 			{
-				it->second->sendClose();
+				if(it->second != nullptr)
+					it->second->sendClose();
```

In all (or some) of these nullptr cases, do we want to throw an exception? It seems that currently there will be no user feedback if the pointer is null.

---

Maybe open an issue for consideration. The problem was that a null de-reference happened when connecting multiple OM producers. (A middle-ground approach would be to sent a `TLOG(TLVL_WARNING)` that the ptr was nullptr unexpectedly.)